dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```diff
@@ -163,7 +164,12 @@ private void cacheInit() {
             if (trace && tc.isDebugEnabled())
                 tcInvoke(cacheStoreService.tcCacheManager, "getCache", metaCacheName, "String", "ArrayList");
```

Comment:
The tcInvoke trace points are intended to capture all operations to the JCache driver, so this, as well as the corresponding tcReturn ought to be included within the 'if' block such that they are only ever printed if we are actually invoking the operation on the driver.  Otherwise, the trace becomes misleading.